src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js

```diff
@@ -50,9 +50,32 @@ export default function AdvisoryDashboard({ page: { setError } }) {
     let parkIdQuery =
       selectedParkId > 0 ? `&protectedAreas.id=${selectedParkId}` : "";
 
-    const { data } = await cmsAxios.get(
-      `/public-advisories?_publicationState=preview&_sort=updated_at:DESC${parkIdQuery}`
-    );
+    const response = await Promise.all([
+      cmsAxios.get(`/management-areas?_limit=-1&_sort=managementAreaName`),
+      cmsAxios.get(
+        `/public-advisories?_publicationState=preview&_sort=updated_at:DESC${parkIdQuery}`
+      ),
+    ]);
+
+    const managementAreas = response[0].data;
+    const publicAdvisories = response[1].data;
+
+    const regionParksCount = managementAreas.reduce((region, item) => {
+      region[item.region.id] = (region[item.region.id] || 0) + 1;
+      return region;
+    }, {});
```

Comment:
shouldn't we have the same logic for "sections" as well?

Reply:
thanks for the feedback... i'll add the sections

Reply:
we are not adding sections for now.
sites will be suffixed to the protected-area name when the information becomes available.